io/FV3GFS_io.F90

```diff
@@ -1084,16 +1077,32 @@ subroutine sfc_prop_restart_read (Sfcprop, Atm_block, Model, fv_domain)
 
         if (Sfcprop(nb)%lakefrac(ix) > zero) then
           Sfcprop(nb)%oceanfrac(ix) = zero ! lake & ocean don't coexist in a cell
-!         if (Sfcprop(nb)%fice(ix) < Model%min_lakeice) then
-!            Sfcprop(nb)%fice(ix) = zero
-!            if (Sfcprop(nb)%slmsk(ix) == 2) Sfcprop(nb)%slmsk(ix) = 0
-!         endif
+          if (Sfcprop(nb)%slmsk(ix) /= one) then
+            if (Sfcprop(nb)%fice(ix) >= Model%min_lakeice) then
+              if (Sfcprop(nb)%slmsk(ix) < 1.9_r8)      &
+                write(*,'(a,2i3,3f6.2)') 'reset lake slmsk=2 at nb,ix=' &
```

Comment:
I am curious how many print lines we will get from the write statement here? Do we need them?

Reply:
Yes, I have confirmed in my own experiments that ice fractionas over lakes are maintained after this fix.
The motivation that lead to finding this bug was to understand why ice concentration over lakes vanished in the coupled runs while the initial conditon had non zero values.

Reply:
Wrong place, please ignore the above comment.

Reply:
We may need to consider commenting these print lines in future commit as there might be too many lines in the log file.